lib/vagrant-service-manager/services/open_shift.rb

```diff
@@ -60,6 +63,30 @@ def self.print_info(ui, options)
         end
       end
 
+      def self.install_cli(machine, ui, options)
+        Installer.new('openshift', machine, ui, options).run
```

Comment:
instead of string for the services, should you not use symbols for our known services? :docker, :openshift, :kubernetes